src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int GROW = 2;
```

Comment:
![image](https://user-images.githubusercontent.com/32339836/216831924-7fcd7209-cfbf-4641-957b-0ff58425e444.png)
```suggestion
    private static final int GROW_FACTOR = 1.5;
```